tests/unit/serializer.test.ts

```diff
@@ -0,0 +1,17 @@
+import Elder, { Serializer, Model, type } from '../../src'
+const config = { adapters: { default: { database: 'ash' } } }
+
+describe('Serializers', () => {
+  test('Serializer base class serialize() static method', () => {
+    const serializer = Serializer.create()
+    class Cat extends Model {
+      @type('string') fizz: string
+    }
+    const orm = Elder.create({ config, models: { cat: Cat } })
```

Comment:
I don't really like that I needed include this call here for this test. The reason for it is that when you create an instance of `Elder` it performs a model setup. I'm feeling tempted to move the setup process to the model such that in this test I could call `Cat.setup(...)` and creating an instance of `Elder` would just iterate over each `Model` and call `Model.setup(...)`